### DISCUSSION TITLE
"Analog/Digital Transition Start" dosn't work (?)

### DISCUSSION THREAD

**Discussion by RasputinBerry** [jomjol/AI-on-the-edge-device#3022] [General]
For weeks I have the problem that the last digital digit changes very early, already at around 0.2 analog. 
The raw value is ALWAYS correct, the value and previous value ALWAYS 1m³ too high. 
![Screenshot 2024-04-03 204816](https://github.com/jomjol/AI-on-the-edge-device-docs/assets/56359831/877959b9-72d5-44d2-9f61-b070c4bade34)
I tried values from 2.5 to 9.9 in the "Analog/Digital Transition Start" settings and read the help text.
![Screenshot 2024-04-03 205117](https://github.com/jomjol/AI-on-the-edge-device-docs/assets/56359831/9120c683-f022-408c-8bb6-f8bd903b89f2)
Nothing helps. What to do ?

**Comment by SybexX** [comment]
you should give more information^^
Which version, what are your settings (which model are you using,...)?
It's best to take a picture with all the relevant settings on it, see my example:
![Settings](https://github.com/jomjol/AI-on-the-edge-device/assets/587201/e9742f82-6837-4adf-b247-b9ab529532b7)

**Comment by RasputinBerry** [comment]
Thank you for your interest... here is the missing information... I have been using these settings for about 2 years, actually without any problems (currently trial & error regarding "Transition"). After some update (currently 15.6.0) it started (without me really noticing)
![config](https://github.com/jomjol/AI-on-the-edge-device/assets/56359831/78297ba7-092e-4418-9bdf-8517fe3717b0)

**Comment by SybexX** [comment]
Reduce your “Maximum Rate Value”, which is too high at 1 (1000 liters consumption in 5 minutes)
increase your "Analog/Digital Transition Start" to at least 9
change your models, your selection no longer exists in the current versions, because these are used in the calculation^^
![model](https://github.com/jomjol/AI-on-the-edge-device/assets/587201/d3a3bad1-21ed-48e8-8cf7-a6c9d7a81f61)

**Comment by RasputinBerry** [comment]
... Thanks for the tip.
I updated to 15.7, new image, new ROI's and adjusted the config. So far it's going great!
One more question: What do the numbers mean? I can't find anything in the [instructions](https://jomjol.github.io/AI-on-the-edge-device-docs/Choosing-the-Model/)....
![confignew](https://github.com/jomjol/AI-on-the-edge-device/assets/56359831/a2996fe8-dae9-4e3a-ab59-e6d8444ad258)

**Reply by SybexX** [reply]
I think this is the version of the model or the number of teaching images used
At least you can say that the higher the number, the newer the model

**Comment by RasputinBerry** [comment]
Okay... the main thing is that it works, thanks again
